src/styles/variables.scss

```diff
@@ -0,0 +1,22 @@
+$font-style: Roboto;
+
+$small-font: 14px;
+$medium-font: 18px;
+$large-font: 24px;
+$extraLarge-font: 36px;
+
+$icon-width: 40px;
+$image-width-medium: 125px;
+$image-width-large: 180px;
+
+$primary-color: #FFFFFF;
+$secondary-color: #776E6E;
+$accent-color: #FCC67C;
+$btn-color: #000000;
+
+$primary-backgroundColor: #000000;
+$secondary-backgroundColor: #171717;
+$comment-backgroundColor: rgba(37, 37, 37, 0.5);
```

Comment:
change it for hex (for example $comment-backgroundColor would be: `#25252580`)

Reply:
done